sites/example-project/src/components/viz/Chart.svelte

```diff
@@ -97,6 +97,9 @@
 
 	export let stacked100 = false;
 
+	export let chartAreaHeight = 180; // Default can be removed when defaults are set by each chart type component
+	chartAreaHeight = Number(chartAreaHeight);
```

Comment:
What is the behavior of this component if the chartAreaHeight is passed as an invalid number? If this errors it could break the rendering of the page.

Reply:
Good call - I'll improve this.